draft-ietf-rats-corim.md

```diff
@@ -2070,82 +2042,147 @@ Logically, new ECT entries are appended to the existing ACS.
+The comparison MUST return true if the value of the SVN in the candidate entry is equal to the value in the condition ECT.
 
-If the Reference value for `measurement-values-map` key 1 is a UINT tagged with
-#6.553 then a minimum comparison is performed. If the value of the SVN in
-ACS less than the value in the Reference Value then the
-Reference Value does not match.
+If the condition ECT value for `measurement-values-map` key 1 is a `uint` tagged with #6.553 then a minimum comparison is performed.
+The comparison MUST return true if the value of the SVN in the candidate entry is less than the value in the condition ECT.
 
 ##### Comparison for digests entries {#sec-cmp-digests}
 
-The value stored under `measurement-values-map` key 2,
-or a value tagged with
-#6.TBD is a digest entry.
-It contains one or more digests, each measuring the
-same object. A Reference Value may contain multiple digests, each with a
-different algorithm acceptable to the Reference Value provider. If the
-digest in Evidence contains a single value with an algorithm and value
-matching one of the algorithms and values in the Reference Value then it
-matches.
+It contains one or more digests, each measuring the same object.
+When multiple digests are provided, each represents a different algorithm acceptable to the condition ECT author.
+
+In the simple case, a condition ECT digests entry containing one digest matches matches a candidate entry containing a single entry with the same algorithm and value.
 
-To prevent downgrade attacks, if there are multiple algorithms which are in
-both the Evidence and Reference Value then the digests calculated using all
-shared algorithms must match.
+To prevent downgrade attacks, if there are multiple algorithms in common between the condition ECT and candidate entry, then the bytes paired with common algorithms must be equal.
+A Verifier SHALL treat two algorithm identifiers as equal if they have the same deterministic binary encoding.
+If both an integer and a string representation are defined for an algorithm then entities creating ECTs SHOULD use the integer representation.
+If condition ECT and ACS entry use different names for the same algorithm, and the Verifier does not recognize that they are the same, then a downgrade attack is possible.
 
-If the CBOR encoding of the `digests` entry in the Reference Value or the
-ACS value with the same key is incorrect (for example if fields
-are missing or the wrong type) then the Reference Value does not match.
+The comparison MUST return false if the CBOR encoding of the `digests` entry in the condition ECT or the ACS value with the same key is incorrect (for example if fields are missing or the wrong type).
 
-The Verifier MUST iterate over the Reference Value `digests` array, locating
-hash algorithm identifiers that are present in the Reference Value and
-in the ACS entry.
+The comparison MUST return false if the condition ECT digests entry does not contain any digests.
```

Comment:
```suggestion
The comparison MUST return false if the condition ECT digests entry does not contain any digests with an algorithm in common with the candidate entry.
```
The `digests-type` is already non-empty, so I assume you mean that there has to be at least 1 matching algorithm to compare digests of.

Reply:
You say this below, so given that non-emptiness is required, I'd say delete the line.

Reply:
The original text seems best IMO.

Reply:
There are conditions below which check for algorithms in common. I would prefer to keep the original text.